core/templates/dev/head/pages/admin-page/misc-tab/admin-misc-tab.directive.html

```diff
@@ -11,7 +11,7 @@ <h3>Topic Similarities</h3>
 
 <md-card class="oppia-page-card oppia-long-text" style="max-width: 700px">
   <h3>Search Index</h3>
-  <button ng-click="$ctrl.clearSearchIndex()">Clear Search Index</button>
+  <button style="max-width: 30%;" ng-click="$ctrl.clearSearchIndex()">Clear Search Index</button>
```

Comment:
Maybe add a class and `<style></style>` at the end of the file?

Reply:
@NishealJ, ping?

Reply:
Has been done Thanks!